### PR TITLE
[stable/mariadb] Support for extraInitContainers like in other popular charts

### PR DESCRIPTION
**Pull request by alokrajiv**
Signed-off-by: alokrajiv <mail@alokrajiv.com>

#### Which issue this PR fixes

This is for a feature request issue: #10421 [Support for extraInitContainers ](https://github.com/helm/charts/issues/10421)

#### Special notes for your reviewer:
My changes are with reference to the same feature in [mysql] (https://github.com/helm/charts/tree/master/stable/mysql#configuration) official helm chart from same repo.

Do note, this feature is available in other charts like elasticsearch, nginx-ingress and mattermost-team-edition. Do, let me know if there is something that is off, i can push the fix.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/mariadb/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: mariadb
-version: 5.3.1
+version: 5.3.2
```
Comment (tompizmor) [inline stable/mariadb/Chart.yaml]: Following the semver versioning, I would have increased the MINOR version as you are adding a new feature:

> Given a version number MAJOR.MINOR.PATCH, increment the:
>
> MAJOR version when you make incompatible API changes,
> MINOR version when you add functionality in a backwards-compatible manner, and
> PATCH version when you make backwards-compatible bug fixes.

```suggestion
version: 5.4.0
```

**Review by tompizmor — CHANGES_REQUESTED** [review]
Can you please apply the same changes from `values.yaml` to `values-production.yaml`.

Also, it would be nice if you could add a new section to the README with an example of use.

File: stable/mariadb/Chart.yaml (hunk 1 above)
Comment (alokrajiv) [reply stable/mariadb/Chart.yaml]: sure, would be happy to :)
thanks for the semver reminder

**Comment by alokrajiv** [comment]
hi @tompizmor,

- [x] added changes to `values-production.yaml`
- [x] Chart Version bumped MINOR
- [x] New Section in README.md with example

done :) ... have a look

**Comment by tompizmor** [comment]
Thanks for the changes @alokrajiv!!

**Review by tompizmor — APPROVED** [review]

**Comment by alokrajiv** [comment]
with pleasure @tompizmor !

**Pull request merged**